src/commands/run.js

```diff
@@ -231,26 +236,25 @@ exports.handler = function ({
         {
           title: 'Download wrapper',
           task: (ctx, task) => {
-            const WRAPPER_PATH = `${os.homedir()}/.aragon/wrapper-${WRAPPER_COMMIT}`
-            ctx.wrapperPath = WRAPPER_PATH
+            clientVersion = clientVersion || defaultClientVersion
+            const CLIENT_PATH = `${os.homedir()}/.aragon/wrapper-${clientVersion}`
+            ctx.wrapperPath = CLIENT_PATH
 
             // Make sure we haven't already downloaded the wrapper
-            if (fs.existsSync(path.resolve(WRAPPER_PATH))) {
+            if (fs.existsSync(path.resolve(CLIENT_PATH))) {
               task.skip('Wrapper already downloaded')
               ctx.wrapperAvailable = true
               return
             }
 
             // Ensure folder exists
-            fs.ensureDirSync(WRAPPER_PATH)
+            fs.ensureDirSync(CLIENT_PATH)
 
             // Clone wrapper
-            return clone(
-              'https://github.com/aragon/aragon',
-              WRAPPER_PATH,
-              { checkout: WRAPPER_COMMIT }
-            )
-          },
+            return clone('https://github.com/aragon/aragon', CLIENT_PATH, {
+              checkout: clientVersion
+            })
```

Comment:
If the checkout fails because an invalid version is passed down to this command, CLI will fail with

```
'git checkout' failed with status code 1
```

I thought it'd be nicer to throw a more descriptive error message, letting the user know that the given version probably doesn't exist, however, hooking into `clone()`'s callback to rethrow the error, results in TaskList actually finishing its tasks successfully, with the error being throw slightly later (as it's async).

This is very likely related to how Listr expects errors to be thrown but needs more investigation.